scripts/create-package/plop-templates-react/tsconfig.json

```diff
@@ -15,8 +15,9 @@
     "moduleResolution": "node",
     "preserveConstEnums": true,
```

Comment:
A couple lines up in this file, can you please remove the `"experimentalDecorators": true` option? (Same in plop-templates-node/tsconfig.json.) It shouldn't be needed for new packages.